swiss_locator/gui/maptip.py

```diff
@@ -22,14 +22,20 @@
 """
 
 from PyQt5.QtCore import Qt, QPoint, pyqtSignal
-from PyQt5.QtWebKit import QWebSettings
-from PyQt5.QtWebKitWidgets import QWebView, QWebPage
 from PyQt5.QtWidgets import QSizePolicy, QDockWidget
 from PyQt5.QtGui import QPalette, QDesktopServices, QCloseEvent
 from qgis.core import Qgis, QgsPointXY, QgsMessageLog
 from qgis.gui import QgisInterface
 
 from swiss_locator import DEBUG
+from swiss_locator.gui.qtwebkit_conf import with_qt_web_kit
+if with_qt_web_kit():
+    from PyQt5.QtWebKit import QWebSettings
+    from PyQt5.QtWebKitWidgets import QWebView, QWebPage
+else:
+    from swiss_locator.core.settings import Settings
+    settings = Settings()
+    settings.set_value("show_map_tip", False)
```

Comment:
why do we need to play with the setting at all?
I wonder if there is something wrong in the current code, `show_map_tip` doesn't seem used?

Reply:
Just playing safe there to handle one edge case.

If for any reason a user has a `show_map_tip` setting with `True` value (e.g., she used the plugin, activated the setting and then uninstalled the Python's QtWebKit package), we'll set ourselves that setting to `False`, so that `show_map_tip()` is never called.

The setting is visited here:
 + https://github.com/opengisch/qgis-swiss-locator/blob/585731ff5c372a8e6e8a2d313f882afb93d817b3/swiss_locator/core/filters/swiss_locator_filter.py#L431
 + https://github.com/opengisch/qgis-swiss-locator/blob/585731ff5c372a8e6e8a2d313f882afb93d817b3/swiss_locator/core/filters/swiss_locator_filter.py#L528

Reply:
let's never set the setting to False, and do `if self.settings.value("show_map_tip") and with_qt_web_kit()` ?

Reply:
Sounds good. 
If we do so, then it wouldn't be worth disabling the map tip checkbox and setting an explanatory tooltip. I'll remove those parts as well.

Reply:
nope, it sounds good to have them disabled since it's something you cannot use anyway

Reply:
Done, I squashed the commits to keep the code base tidy.